### PR TITLE
[DO NOT MERGE] Added a comment

### PR DESCRIPTION
**Pull request by hahnbeelee**
### Summary 
Just showing Michael and Noah a PR lol

Remaining TODOs:

- [ ] resolve bug 1
- [ ] implement Z

Depends on #{number of PR}

### Blockers 

- A newly discovered dependency that hasn’t been addressed

### Breaking Changes  

- Database schema change (anything that changes Firestore collection structure)
- Other change that could cause problems (Detailed in notes)

### CODE REVIEW

File: src/requirements/data/index.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ import spanishMinorRequirements from './minors/spanish';
 
 // import mengCSRequirements from './grad/meng-cs';
 import mpaRequirements from './grad/mpa';
-
+// requirements!
```
Comment (hahnbeelee) [inline src/requirements/data/index.ts:71]: bad

**Review by hahnbeelee — COMMENTED** [review]
bug

**Review by noschiff — COMMENTED** [review]
good stuff

**Comment by hahnbeelee** [comment]
fdsafsda

File: src/requirements/data/index.ts [hunk 2]
```diff
@@ -68,7 +68,7 @@ import spanishMinorRequirements from './minors/spanish';
 
 // import mengCSRequirements from './grad/meng-cs';
 import mpaRequirements from './grad/mpa';
```
Comment (noschiff) [inline src/requirements/data/index.ts:70]: hmm

**Review by noschiff — CHANGES_REQUESTED** [review]
idk

**Review by noschiff — APPROVED** [review]
ok

**Pull request closed without merging**